MCprep_addon/conf.py

```diff
@@ -19,6 +19,19 @@
 import os
 from pathlib import Path
 import bpy
+from bpy.utils.previews import ImagePreviewCollection
+
+from typing import Union, Literal, Tuple, List, Dict
+from mathutils import Vector
+
+VectorType =  Union[Tuple[float, float, float], Vector]
+
+Form = Literal["mc", "mineways", "jmc2obj"]
+# Eevee Next now exists as an engine option in Blender code
```

Comment:
I would suggest changing the comment to the following. It better justifys the reason for having `BLENDER_EEVEE_NEXT` as an option.
```suggestion
# EEVEE Next is the next generation EEVEE. So in preperation for that, 
# we've added "BLENDER_EEVEE_NEXT" as an Engine option
```